pom.xml

```diff
@@ -51,6 +51,7 @@ under the License.
         <bundle.snappy>false</bundle.snappy>
         <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
         <hadoop.version>2.6.0</hadoop.version>
+        <hadoop-core.version>1.2.1</hadoop-core.version>
```

Comment:
From maven centrol repo [1], the latest hadoop-core version is 1.2.1.

[1] http://search.maven.org/#search%7Cgav%7C1%7Cg%3A%22org.apache.hadoop%22%20AND%20a%3A%22hadoop-core%22